package.json

```diff
@@ -116,12 +116,6 @@
     "last 2 Edge versions",
     "last 2 Safari major versions"
   ],
-  "husky": {
```

Comment:
Please put this back. The issue you had is because of Windows. Please see https://github.com/AlaskaAirlines/auro-popover/blob/master/package.json#L105

Reply:
I'm not sure how to fix this.   I can't commit with this line.
![image](https://user-images.githubusercontent.com/5118118/108809022-d0798d00-755c-11eb-8ea2-dc51b2be4813.png)

Reply:
Line 97 `"cssLint": "stylelint './src/style.css'",`

Update to `"cssLint": "stylelint \"./src/*.css\"",` as I linked to in the previous comment. That will address the issue with cssLint. 

Windows can't find the file because it doesn't understand the path.

Reply:
Sweet thanks for the added context.